src/Client.php

```diff
@@ -220,4 +233,11 @@ private function setFlags(Horde_Imap_Client_Search_Query $hordeQuery,Query $quer
             }
         }
     }
+    /**
+     * 
+     * @param boolean $peek
+     */
+    public function setPeek($peek) {
+        $this->peek = $peek;
```

Comment:
The `$this->peek` variable is set but it seems it's never read?

Reply:
Yeah sure for some reasons the read method was not in the version i pushed.